### PR TITLE
Prepare Release/2.2.0

### PR DESCRIPTION
**Pull request by Rahmon**
### Checklist:

- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [2.2.0] - 2021-08-09
+
+### Breaking Changes
+- BC_Utility API changed:
+  - See `set_cache_item` and `delete_cache_item` in `includes/class-bc-utility.php`.
+  - Removed `remove_deleted_players` function.
```
Comment (felipeelia) [inline CHANGELOG.md:12]: Don't we want to give credits in this section @Rahmon ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file, per [the Ke
...
+
+### Breaking Changes
+- BC_Utility API changed:
+  - See `set_cache_item` and `delete_cache_item` in `includes/class-bc-utility.php`.
+  - Removed `remove_deleted_players` function. 
+### Fixed
+- Fix: undefined index warnings. Props [@sanketio](https://github.com/sanketio).
+- Fix: typo for the `$allowedtags` global used in conjunction with wp_kses. Props [@theskinnyghost](https://github.com/theskinnyghost).
+- Fix: performance issue related with bc_transient_keys option. Props [@Rahmon](https://github.com/Rahmon)
+- Fix: playlist preview in the editor. Props [@Rahmon](https://github.com/Rahmon).
```
Comment (felipeelia) [inline CHANGELOG.md:17]: Not sure if we need to repeat "Fix" in this section, as all items are under "Fixed" anyway.

File: readme.txt [hunk 3]
```diff
@@ -5,7 +5,7 @@ Donate link: https://supporters.eff.org/donate
 Tags: brightcove, 10up, videos, video
 Requires at least: 4.2
 Tested up to: 5.7
```
Comment (felipeelia) [inline readme.txt:7]: Perhaps a good opportunity to bump the WP version as well? Also, README.md still has it as 5.6.

File: CHANGELOG.md (hunk 1 above)
Comment (Rahmon) [reply CHANGELOG.md:12]: I'd say no. This is related to an item in the Fixed section and I give credits there.

**Review by felipeelia — APPROVED** [review]

**Pull request merged**